client/src/index.tsx

```diff
@@ -58,7 +58,16 @@ if (window.location.hash) {
   ReactDOM.createRoot(root).render(
     <WagmiProvider config={wagmiConfig}>
       <QueryClientProvider client={clientReactQuery}>
-        <RainbowKitProvider modalSize="compact">
+        <RainbowKitProvider
+          modalSize="compact"
+          theme={lightTheme({
+            accentColor: '#2d9b87',
```

Comment:
You can use exported values from SCSS (like in `mediaQueries.ts`).

Reply:
We have more of these. I created https://linear.app/golemfoundation/issue/OCT-1925/export-colors-from-scss-to-ts-file to handle that.